### DISCUSSION TITLE
What is the recommended way to deploy goakt in multi region?

### DISCUSSION THREAD

**Discussion by sdil** [Tochemey/goakt#411] [General]
Multi-region communication is hard because the latency will be too high for actor-to-actor communication. What is the recommended way to deploy to multiple regions?

**Comment by Tochemey** [comment]
Honestly I have not looked into that yet. I know Akka and Apache Pekko have the concept of multi-dc but never bother to look at it.

**Reply by Tochemey** [reply]
@sdil I can create an issue for this. I hope to have the bandwith to look into it.

**Reply by sdil** [reply]
Thanks @Tochemey I'm honestly not too sure how multi dc clustering works in Pekko & Akka either.

**Comment by sdil** [comment]
I was doing some research and wish to share here. I saw that Cloudflare Durable Object will stick to 1 location when it's created and do not currently change locations after they are created. This sounds like it's a single region deployment to me. I guess it's not common for actor model program to be deployed to multiple regions.

Ref: https://developers.cloudflare.com/durable-objects/reference/data-location/#provide-a-location-hint

**Comment by Tochemey** [comment]
@sdil I will close this issue. We can always look into it if the need arises.

**Reply by sdil** [reply]
Sure!
